packages/render-fidelity-tools/package.json

```diff
@@ -31,9 +31,10 @@
     "@types/resize-observer-browser": "^0.1.2",
     "@types/rimraf": "^2.0.3",
     "@webcomponents/webcomponentsjs": "~2.1.3",
-    "filament": "1.4.5",
+    "filament": "^1.5.0-rc.1",
```

Comment:
How long until you folks graduate this package from `rc`?

Reply:
Probably in a week or two, we were going to wait until we land our upcoming variance shadow maps feature... unless something comes up that motivates an earlier release.